<launch>


    <arg name="cloud_1" default="/ti_mmwave_front_radar/radar_scan_pcl"/>
    <arg name="cloud_2" default="/ti_mmwave_left_radar/radar_scan_pcl"/>
    <arg name="cloud_3" default="/ti_mmwave_right_radar/radar_scan_pcl"/>
    <arg name="cloud_4" default="/ti_mmwave_rear_radar/radar_scan_pcl"/>

    <node name="async_cloud_concatenator" pkg="cloud_concatenator" type="async_concatenator_node" output="screen">
        <remap from="cloud1"       to="$(arg cloud_1)"/>
        <remap from="cloud2"       to="$(arg cloud_2)"/>
        <remap from="cloud3"       to="$(arg cloud_3)"/>
        <remap from="cloud4"       to="$(arg cloud_4)"/>
        <param name="frequency"    value="10"/>
        <param name="target_frame" value="base_link"/>
    </node>


</launch>